mmv1/products/compute/api.yaml

```diff
@@ -5733,10 +5733,11 @@ objects:
                       name: 'sourceImage'
                       description: |
                         The source image to create this disk. When creating a
-                        new instance, one of initializeParams.sourceImage or
-                        disks.source is required.  To create a disk with one of
-                        the public operating system images, specify the image
-                        by its family name.
+                        new instance, one of initializeParams.sourceImage,
```

Comment:
Can you revert the changes to instance template here? We're likely to remove its definitions entirely in the near future so I'd like to avoid modifications.